src/mapping_navigation/scripts/pure_pursuit.py

```diff
@@ -60,39 +63,25 @@ def __init__(self, look_ahead_distance: float, look_forward_gain: float, path :L
         self.look_forward_gain = look_forward_gain
         self.look_ahead_distance = look_ahead_distance
         self.path = path
-        #Needs to be refactored to look at our points instead of his cx and cy
         self.old_nearest_point_index = None
         self.old_steer_target = 0
-        self.cx = []
-        self.cy = []
-        for point in path:
-            self.cx.append(point.coordinate[0])
-            self.cy.append(point.coordinate[1])
 
     """
     Returns the steering angle in radians and the index to the goal points its turning to
     Takes in the current state of the car
     """
     def pure_pursuit_steer_control(self, state: State) -> Tuple[float, int]:
         
-        # Getting the next point
         ind, Lf = self.search_target_index(state)
```

Comment:
If you changed the member variable of Navigation from "target_ind" to "target_index" (mentioned in a different comment), maybe rename this variable as well for consistency.

Lf is a pretty condensed name, can we name it to something with more meaning?

Reply:
agreed, will change